### DISCUSSION TITLE
Managing User PATs and Team Tokens

### DISCUSSION THREAD

**Discussion by ben-balk-oc** [terrakube-io/terrakube#684] [Q&A]
Is there a way to revoke or delete API tokens for users or teams? I'm not seeing any way to do this through the UI or API. 

It would be helpful to be able to revoke tokens when a new one is generated. Ideally, it would be useful to provide a way for a user to clean up their own tokens in the UI but a good start would be an API method that would allow a DELETE of token resources by ID.

Along the same line, it would be nice to delete old/expired tokens just to keep things tidy on the User Settings page.

**Comment by alfespa17** [comment]
Hello @ben-balk-oc there is no way to revoke user or teams tokens right now, that will be a new feature request

**Reply by alfespa17** [reply]
I created a new feature request in the following issue: https://github.com/AzBuilder/terrakube/issues/685
